components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/AbstractHTTPFunction.java

```diff
@@ -218,4 +333,21 @@ protected void setHeaders(HttpUriRequest request, Map<String, String> headers) {
                 .filter(entry -> StringUtils.isNotBlank(entry.getKey()) && !entry.getKey().equals("null"))
                 .forEach(entry -> request.setHeader(entry.getKey(), entry.getValue()));
     }
+
+    /**
+     * Get AuthConfigModel from the map.
+     *
+     * @param map Map of properties.
+     * @return AuthConfigModel.
+     */
+    protected AuthConfigModel getAuthConfigModel(Map<String, Object> map) {
+        AuthConfigModel authConfig;
+        if (map.get("type") == null || map.get("properties") == null) {
+            throw new IllegalArgumentException("Invalid argument type. Expected AuthConfigModel.");
+        }
```

Comment:
This exception should include the expected format

Reply:
Fixed with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/05c4f77e8d18ba911c3b98bc7e18ec3fb73dcd33